app/src/main/java/fr/gaulupeau/apps/Poche/ui/ReadArticleActivity.java

```diff
@@ -513,6 +515,10 @@ private void showDisableTouchToast() {
     @SuppressLint("SetJavaScriptEnabled")
     private void initWebView() {
         webViewContent.getSettings().setJavaScriptEnabled(true);
+        if(settings.isMathRenderingEnabled()) {
+            // This is needed to avoid CORS-errors for the bundled katex-scripts
+            webViewContent.getSettings().setAllowUniversalAccessFromFileURLs(true);
```

Comment:
I'm still not sure about this.

Reply:
I think dropping `crossorigin="anonymous"` in `katex_part.html` should do it.

What's the version of Android you get this error with?

Reply:
I was getting the error on Android 10. You are right though, dropping the `crossorigin="anonymous"` fixes this as well for me.